src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3176,6 +3176,19 @@ operand immL_32bits()
   interface(CONST_INTER);
 %}
 
+// Int Immediate: power of 2, positive
+operand immI_bitmask()
+%{
+  predicate((n->get_int() != 0)
+            && ((n->get_int() & 0xc0000000) == 0)
```

Comment:
Why do you use `& 0xc0000000` and not `& 0x80000000` (or even `n->get_int() > 0`)?

Reply:
Yes. I believe `n->get_int() > 0` would work and be simpler to read.